### ISSUE TITLE
DOTCOM-25314 - Accessibility: html lang attribute is missing

### ISSUE THREAD

**Issue by rofe** [adobe/theblog#332]
From: https://jira.corp.adobe.com/browse/DOTCOM-25314
URL: https://blog.adobe.com/

**Expected behavior**

When configuring a screen reader, users select a default language. If the language of a webpage is not specified, the screen reader assumes the default language set by the user. Language settings become an issue for users who speak multiple languages and access website in more than one language. It is essential to specify a language and ensure that it is valid so website text is pronounced correctly.

Screen readers use different sound libraries for each language, based on the pronunciation and characteristics of that language. Screen readers can switch between these language libraries easily, but only if the documents specify which language(s) to read and when. If the language is not specified, the screen reader reads the document in the user's default language, resulting in a strange accent! It is impossible to understand anything when screen readers are using the wrong language library.

Add lang attribute.

Example: `<html lang="en"> ... </html>`

Refer : https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H57

**WCAG Success Criteria**

[Language of Page](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-doc-lang-id.html)

**Comment by rofe** [comment]
Requires https://github.com/adobe/helix-pages/pull/385
